invenio_app_rdm/communities_ui/views/ui.py

```diff
@@ -64,23 +74,19 @@ def create_ui_blueprint(app):
     )
 
     blueprint.add_url_rule(
-        routes["community-home"],
-        view_func=communities_home,
+        **create_url_rule(routes["community-home"], communities_home)
```

Comment:
minor: There's a `community-detail` route above that has not been converted to the configurable style of `create_url_rule`